modules/installation-creating-gcp-vpc.adoc

```diff
@@ -27,9 +27,6 @@ have to contact Red Hat support with your installation logs.
 .Prerequisites
 
 * Configure a GCP account.
```

Comment:
1. In fact, "Creating a VPC in GCP" is not the first section which needs such prerequisites, the section "Creating the installation configuration file" should be the first. 
2. Besides, I'm not quite sure the exact meaning of "Configure a GCP account". If it means to activate the GCP service account (configured during "Creating a service account in GCP"), suggest to also configure the GCP project, where to deploy OpenShift cluster.

Reply:
Besides, a question on the whole logic. Thanks in advance! 

Supposing a previous section had mentioned an item as a "Prerequisites", the expected behavior now is that, the current section can ignore the same item in its "Prerequisites", right? 

If so, what if there is a previous section explaining the item specifically, for example, the section "Creating a VPC in GCP" explains what and how to create VPC resources, shall the subsequent section "Creating load balancers in GCP" still list "Create and configure a VPC and associated subnets in GCP" as its "Prerequisites"?